PaintingTrivia.py

```diff
@@ -1,9 +1,23 @@
 import random
-from flask import request, session, render_template, Markup, redirect, url_for, Flask
+from flask import request, session, render_template, Markup, redirect, url_for, Flask, flash
 from sqlalchemy.sql.functions import func
 from models.painter import Painter
 from models.painting import Painting
+from models.user import User
 from settings import app
+from database import users
+from settings import db
+
+
+def layout_buttons(request):
+    print request.form.values(), request.__dict__.keys()
+    print request.form
+    if 'Login' in request.form.values():
```

Comment:
i don't like it:
1) document what layout buttons means (it's not clear)
2) why not save an action in a 'hidden' form field and check 

```request.form['navigate_to'] in ['Login','Register', 'Logout']:
      return redirect(url_for(request.form['navigate_to'])
```